x/gamm/keeper/gas_test.go

```diff
@@ -74,8 +74,6 @@ func (s *KeeperTestSuite) TestJoinPoolGas() {
 	))
 
 	firstJoinGas := s.measureJoinPoolGas(defaultAddr, poolId, minShareOutAmount, defaultCoins)
-	// UNFORKINGNOTE: This used to be capped at LessOrEqual to 100000, but unforking increased this value.
```

Comment:
Gas change is due to logic change in bank module (bank send)

Reply:
Do you know what extra logic is done in bank sends that causes this increase?

Reply:
I don't I can specifically debug and track down which causes this will get back to you

Reply:
@mattverse any updates here?

Reply:
Apologies for the delay here, https://github.com/osmosis-labs/cosmos-sdk/blob/osmo/v0.50.x/x/bank/keeper/send.go#L246-L250 it's this specific logic that got added to send